packages/account/src/account.ts

```diff
@@ -508,6 +508,22 @@ export class Account extends AbstractAccount {
     return this._connector.signMessage(this.address.toString(), message);
   }
 
+  /**
+   * Signs a transaction with the wallet's private key.
+   *
+   * @param transactionRequestLike - The transaction request to sign.
+   * @returns A promise that resolves to the signature of the transaction.
+   */
+  async signTransaction(hashedTransaction: TransactionRequestLike): Promise<string> {
```

Comment:
```suggestion
  async signTransaction(transactionRequestLike: TransactionRequestLike): Promise<string> {
```

to match the doc blocks

Reply:
Good spot

Reply:
🦅  👀